src/mbgl/tile/geometry_tile.hpp

```diff
@@ -28,6 +28,8 @@ class TileAtlasTextures;
 
 class GeometryTile : public Tile, public GlyphRequestor, public ImageRequestor {
 public:
+    const std::thread::id renderThreadID = std::this_thread::get_id();
```

Comment:
Nit: this variable name might be accurate now, but it won't be if this class is ever used off the render thread.

Reply:
If it's ever created off the render thread, yes.  But in that case I expect it would be removed entirely because the checks against it wouldn't make sense anymore.